src/components/KakaoLogin.tsx

```
@@ -0,0 +1,49 @@
+'use client';
+
+import Script from 'next/script';
+
+import Button from '@/components/ui/Button';
+
+export default function KakaoLogin() {
+  const KAKAO_SDK_URL = 'https://t1.kakaocdn.net/kakao_js_sdk/2.7.1/kakao.min.js';
+  const KAKAO_JAVASCRIPT_KEY = process.env.NEXT_PUBLIC_KAKAO_JAVASCRIPT_KEY;
+
+  function handleLogin() {
+    if (window.Kakao && window.Kakao.isInitialized()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)